tests/helpers/fake-redis-client.js

```diff
@@ -11,10 +12,12 @@ module.exports = CoreObject.extend({
   set: function() { },
   del: function() { },
   zadd: function(key, score , tag) {
-    this.recentRevisions.push(key + ':' + tag);
+    assert(key.match(/:revisions$/));
```

Comment:
yeah, this is kind of ugly. maybe you can extend the fake client with a method that has the assert for just the test that cares about it?